apps/fpc/fsw/unit_test/fpc_test_utils.cpp

```diff
@@ -313,8 +324,8 @@ double FPC_Test_GetChecksum(FPC_ConfigTbl_t *pTbl)
                + (double)pTbl->FPC_Runway_Takeoff.MAX_PITCH_RADIANS
                + (double)pTbl->FPC_Runway_Takeoff.MAX_ROLL_RADIANS
                + (double)pTbl->FPC_Runway_Takeoff.AIRSPD_SCL
-               + (double)pTbl->FPC_Runway_Takeoff.AIRSPD_MIN
-               + (double)pTbl->FPC_Runway_Takeoff.CLMBOUT_DIFF;
+               + (double)pTbl->FPC_Runway_Takeoff.RWTO_AIRSPD_MIN
+               + (double)pTbl->FPC_Runway_Takeoff.RWTO_CLMBOUT_DIFF;
 
     printf("FPC_Test_GetChecksum:\n");
     std::cout.precision(17);
```

Comment:
You should not mix C++ and C patterns (printf vs std::cout).  Use one or the other.

Reply:
I will use printf from now on.

Reply:
Done